seclea_ai/svc/api/collection/aysnc_dataset.py

```diff
@@ -0,0 +1,60 @@
+import json
+import os
+import aiohttp
+
+from seclea_ai.lib.seclea_utils.core.transmission import Transmission
+
+root = "/collection/datasets"
+
+async def async_post_dataset(
+    transmission: Transmission,
```

Comment:
I would like this to all be in internal/api not here. We will be removing svc when we have completed the refactor.

Reply:
Moved in the latest commit.